src/styles/components/_carousel.scss

```diff
@@ -58,7 +82,7 @@
     border: none;
     background-color: $mine-shaft;
     color: $light;
```

Comment:
For your concern, how about using `outline: thin dotted;` instead since the main purpose of this is for [accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility)

Further reading [here](https://www.a11yproject.com/posts/2013-01-25-never-remove-css-outlines/)